cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfRobotComponent.cpp

```diff
@@ -55,3 +52,31 @@ void UUrdfRobotComponent::createChildComponents(UrdfLinkDesc* parent_link_desc,
         createChildComponents(child_link_desc, child_link_component);
     }
 }
+
+void UUrdfRobotComponent::applyKeyInputs()
+{
+    UInputComponent* InputComponent = GetOwner()->InputComponent;
+
+    for (auto& input_axis_binding : input_axis_bindings_) {
+        float axis_value = InputComponent->GetAxisValue(input_axis_binding.axis_name_);
+
+        switch (input_axis_binding.type_) {
+        case AxisBindingType::Set:
+            if (axis_value > 0) {
+                for (uint32 i = 0; i < input_axis_binding.component_names_.size(); i++) {
```

Comment:
int i = 0 for consistency with our other loops

Reply:
removed as data structure changed.